certification/internal/engine/openshift.go

```diff
@@ -58,7 +72,7 @@ func (pe OpenshiftEngine) CreateNamespace(name string, opts cli.OpenshiftOptions
 	}, nil
 }
 
-func (pe OpenshiftEngine) DeleteNamespace(name string, opts cli.OpenshiftOptions) error {
+func (pe openshiftEngine) DeleteNamespace(name string, opts cli.OpenshiftOptions) error {
```

Comment:
NIT: should this be `oe` instead of `pe`?

Reply:
Done :)